### ISSUE TITLE
Change the way xlabels are specified

### ISSUE THREAD

**Issue by jcrudy** [scikit-learn-contrib/py-earth#21]
The xlabels argument is currently treated as a hyperparameter.  It should probably be treated differently.  Issues:
1. In the Pandas case, the labels come with the data and should therefore be set in fit.
2. In the Pandas case, column order may be less relevant than column name.

At the very least, I should make it easy to set xlabels in the fit method.
